routemaster/cron.py

```diff
@@ -43,7 +42,7 @@ def _retry_metadata_updates(
     state_machine,
     should_terminate,
 ) -> None:
-    print(f"Processing metadata update trigger for {state.name}")
+    process_gate_metadata_retries(app, state, state_machine, should_terminate)
```

Comment:
Now that we have the action functions which are being called, do we need these indirections at all?

Reply:
0968fcacc13206fdcc945eebf4aa701b17f8a20f

I thought that would be worse to move!